tutorials/inverse/80_brainstorm_phantom_elekta.py

```diff
@@ -49,8 +49,8 @@
 raw.info['bads'] = ['MEG1933', 'MEG2421']
 
 # %%
-# The data have strong line frequency (60 Hz and harmonics) and cHPI coil
-# noise (five peaks around 300 Hz). Here we plot only out to 60 seconds
+# The data has strong line frequency (60 Hz and harmonics) and cHPI coil
```

Comment:
I'm not sure whether or not you actually need to / should change these -- at some point I learned `data` was plural of `datum`, but I have also learned to live with people never using it that way 🤷

Reply:
It sounds weird to me, but I'm not a native English speaker :sweat_smile: